twinkle.js

```diff
@@ -27,7 +27,6 @@ if (!Morebits.userIsInGroup('autoconfirmed') && !Morebits.userIsInGroup('confirm
 var Twinkle = {};
 window.Twinkle = Twinkle;  // allow global access
 
-// for use by custom modules (normally empty)
 Twinkle.initCallbacks = [];
 Twinkle.addInitCallback = function twinkleAddInitCallback(func) {
```

Comment:
Is it not a problem to reuse this?  This basically just changes the standard loading system to the one for "custom modules" afaict, but since we've got `Twinkle.addInitCallback = function` twice now I would think this would break the "custom modules".

Reply:
I don't think there's a problem as no such "custom modules" exist. Twinkle.addInitCallback is completely unused (only search results are old copies of Twinkle code).

Reply:
@siddharthvp Okay but from a code point of view, it should be renamed or the former call be removed.  I *think* it's used on other projects, so a simple rename of this new function should suffice?